backend/src/db-utils.js

```diff
@@ -240,6 +240,24 @@ const queryDashboardTransactionsStats = async (options) => {
   };
 };
 
+const queryReceiptsList = async (transactionHash) => {
```

Comment:
leave the comment at the frontend/src/libraries/explorer-wamp/transactions.ts file